### DISCUSSION TITLE
Compatibility Matrix Discussion - What robot do you use with this library?

### DISCUSSION THREAD

**Discussion by funkybunch** [sharkiqlibs/sharkiq#29] [Ideas]
I'd like to start assembling a compatibility matrix to add to the Home Assistant documentation.  I assume since the robots use the Ayla API, that this library is likely compatible with all of them, but I think providing specific models that we've been using will also be helpful to people in the docs.

I'm using an AV2001.  What is everyone else using?  Are there any features that _don't_ work on your robot?

cc other maintainers & active contributors:
@JeffResc @TheLastFrame @travipross @ElMoselYEE @Mobius5150

**Comment by TheLastFrame** [comment]
My robot is the _RV1100S1EU_ (or RV1100S1EU 32 (?)) at least that's what's written on the device. The bill says it's the _RV1000SEU_. Idk what the difference is. But it's an EU only device (I think) and with a Self-Empty base if that matters.

ADD: The start, stop and the custom room selection feature work for me.

**Reply by funkybunch** [reply]
Did you test these services through home assistant or the Python library?

I'd love to have a few folks test the custom room selection specifically in Home assistant before we release it to the shark integration for general availability.

If you ended up testing in HA did the region selector work?  I get an error when I try to authenticate using the EU servers, which I assume is expected behavior, but I have no way to confirm it is working 😂

**Reply by TheLastFrame** [reply]
Until now I tested it only with the library. I Today I wasn't able to setup a test environment for HA, but I hope that I have time to test this tomorrow.

I will update you here & on the HA PR when I finished testing.

**Reply by funkybunch** [reply]
Awesome - much appreciated!

**Comment by TheLastFrame** [comment]
BTW I think @captrik has the same device.

**Comment by emann3** [comment]
I have two robots on the same Shark account - both Model RV2520A0US
I use this api strictly from the command line.
(Robots are on separate floors, so their room lists are different)
I am able to select a room and send a robot to clean that room.
I haven't tried much else with this api yet - just getting started.  
I want to be able to schedule rooms with my 2nd robot using this api because Google home/assistant will only schedule rooms with the 1st robot.  When it tries to schedule the 2nd robot, the shark service responds with "the robot is offline". 
And of course the shark app only allows scheduling everything - no ability to schedule a room.

@JeffResc @TheLastFrame @travipross @ElMoselYEE @Mobius5150

**Reply by tombrant** [reply]
Is anybody actively testing with this?  I've tried scheduling rooms with Python on command line, but whenever the  SetAreasToClean method is used, it has no effect.  The robot goes out but just starts cleaning the whole house.  Alexa and Google Assistant also are not working with room-specific cleaning.

**Comment by Gr8hifi** [comment]
Does the clean room command work with home assistant?  I’ve tried setting it up in automation but it fails on run.  My guess is it has to do with capitalization.  I can’t make rooms start with a lower case letter in the shark app and I’m not sure how to use uppercase letter in home assistant.  Screen shot of error is below.  

<img width="977" alt="image" src="https://github.com/JeffResc/sharkiq/assets/80473708/0acc502e-d0ff-4f16-bc19-388d14f28717">

**Reply by tombrant** [reply]
Did you ever get past this?  Are you seeing a generalized error during authentication in HA with the sharkiq integration recently.
